Cargo.toml

```diff
@@ -16,17 +16,36 @@ exclude = [
 ]
 
 [dependencies]
-dusk-plonk = {version = "0.8.2", default-features = false, features = ["alloc"]}
-dusk-bls12_381 = "0.8"
-dusk-jubjub = "0.10"
-dusk-schnorr = "0.7.1"
-dusk-poseidon = { version = "0.22", features = ["canon"] }
-dusk-bytes = "0.1"
+rand_core = "0.6.0"
+rand = "0.8"
+thiserror = "1.0.30"
+
+ark-plonk = { version = "^0.8", git = "https://github.com/rust-zkp/ark-plonk", default-features = false }
```

Comment:
Is it possible to update to the new repo? https://github.com/ZK-Garage/plonk

Reply:
yes, updated